salt/metalk8s/kubeadm/init/certs/apiserver.sls

```diff
@@ -0,0 +1,34 @@
+{%- from "metalk8s/map.jinja" import kube_api with context %}
+
+Create kube api private key:
+  x509.private_key_managed:
+    - name: {{ kube_api.cert.public_key }}
+    - mode: 600
+    - makedirs: True
+
+{#- Add FQDN of the server #}
+{%- do kube_api.cert.subjectAltName.get('DNS').insert(0, grains['fqdn']) %}
+
+{#- TODO: Not use all IPs and some logic to get service IP #}
+{%- do kube_api.cert.subjectAltName.get('IP').append(
+         salt['netaddress.list_cidr_ips'](kube_api.service_ip_range)[1]) %}
```

Comment:
Let's set the service IP as a static pillar value instead, which is what it should be anyway, rather than compute it at various places.

Reply:
Ok got my answer for my previous comment :)